tests/core/contracts/conftest.py

```diff
@@ -527,6 +527,52 @@ def FallballFunctionContract(web3, FALLBACK_FUNCTION_CONTRACT):
     return web3.eth.contract(**FALLBACK_FUNCTION_CONTRACT)
 
 
+CONTRACT_RETURN_ARGS_SOURCE = """
+contract CallerTester {
+  function returnMeta() public payable returns (address, bytes memory, uint256, uint256) {
```

Comment:
you might return block number too which would make it easy to test calls at specific block numbers.